### DISCUSSION TITLE
Clarification of Deepdoctection Working

### DISCUSSION THREAD

**Discussion by Dipankar1997161** [deepdoctection/deepdoctection#239] [Q&A]
@ryanmcdonough Thank you for the work.

 Does DeepDoctention only is able to parse tables from PDFs containing only Tables? 

or is it also able to extract the tables from Academic Research papers which has Text, Table, images?  My task is parse an academic paper, so can I use it?

I look forward to hear from you

**Comment by aiorga-sherpas** [comment]
Yes, you can use it.
